modules/pytket-braket/pytket/extensions/braket/backends/braket.py

```diff
@@ -203,21 +203,21 @@ def _obs_from_qpo(operator: QubitPauliOperator, n_qubits: int) -> Observable:
 
 def _get_result(
     completed_task: Union[AwsQuantumTask, LocalQuantumTask],
-    target_qubits: str,
+    target_qubits: List[int],
+    measures: Dict[int, int],
```

Comment:
Can we assume from Braket that qubits will always be single indexed and won't have different strings to describe different subsets of qubits?

Reply:
Good question. No, but it is already the case that `tk_to_braket` assumes all qubits belong to a single, singly-indexed, register. So we are not breaking anything here, but I will see if I can relax that assumption, and if not add a predicate to assert that we only support such circuits.